src/terraform.js

```diff
@@ -101,16 +101,16 @@ resource "iterative_cml_runner" "runner" {
   ${region ? `region = "${region}"` : ''}
   ${type ? `instance_type = "${type}"` : ''}
   ${gpu ? `instance_gpu = "${gpu}"` : ''}
-  ${hdd_size ? `instance_hdd_size = ${hdd_size}` : ''}
-  ${ssh_private ? `ssh_private = "${ssh_private}"` : ''}
+  ${hddSize ? `instance_hdd_size = ${hddSize}` : ''}
+  ${sshPrivate ? `ssh_private = "${sshPrivate}"` : ''}
   ${spot ? `spot = ${spot}` : ''}
-  ${spot_price ? `spot_price = ${spot_price}` : ''}
-  ${startup_script ? `startup_script = "${startup_script}"` : ''}
+  ${spotPrice ? `spot_price = ${spotPrice}` : ''}
+  ${startupScript ? `startup_script = "${startupScript}"` : ''}
```

Comment:
PS do these have to remain `snake_case`? Not sure what `resource iterative_cml_runner` is used for.

Reply:
That's a Terraform template for cloud provisioning, and snake_case should be preserved in the [HCL](https://github.com/hashicorp/hcl) identifiers. The JavaScript counterparts can be dromedaryCase though. It's fine as-is.

We should probably use [Terraform CDK](https://www.hashicorp.com/blog/cdk-for-terraform-enabling-python-and-typescript-support) instead, but that's a separate discussion.